db/db_test2.cc

```diff
@@ -891,8 +894,12 @@ TEST_F(DBTest2, WalFilterTestWithChangeBatch) {
                                   bool* batch_changed) const override {
       if (current_record_index_ >= change_records_from_index_) {
         ChangeBatchHandler handler(new_batch, num_keys_to_add_in_new_batch_);
-        batch.Iterate(&handler);
-        *batch_changed = true;
+        Status s = batch.Iterate(&handler);
+        if (s.ok()) {
+          *batch_changed = true;
+        } else {
+          assert(false);
```

Comment:
Should this be assert(false) or FAIL()?

Reply:
`FAIL` can't be used here which is why I went with `assert(false)`:
```
db/db_test2.cc:901:8: error: cannot initialize return object of type 'rocksdb::WalFilter::WalProcessingOption' with an rvalue of type 'void'
       FAIL();
       ^~~~~~
```